src/App.tsx

```diff
@@ -14,11 +15,37 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+export const App: React.FC = () => {
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button type="button" className="button is-info is-light">
+          Sort alphabetically
+        </button>
 
-export default App;
+        <button type="button" className="button is-success is-light">
+          Sort by length
+        </button>
+
+        <button type="button" className="button is-warning is-light">
+          Reverse
+        </button>
+
+        <button type="button" className="button is-danger is-light">
+          Reset
+        </button>
+      </div>
+
+      <ul>
+        <ul>
```

Comment:
There is a nested `<ul>` inside another `<ul>`, which is not semantically correct. Consider removing the outer `<ul>` or replacing it with a `<div>` to ensure proper HTML structure.